src/atlas/interpolation/method/fe/FiniteElement.cc

```diff
@@ -132,20 +132,26 @@ struct Stencil {
 void FiniteElement::print( std::ostream& out ) const {
     functionspace::NodeColumns src( source_ );
     functionspace::NodeColumns tgt( target_ );
+    if ( mpi::rank() == 0 ) {
+        out << "atlas::interpolation::method::FiniteElement" << std::endl;
+        out << "maxFractionElemsToTry_: " << maxFractionElemsToTry_;
+        out << " treat_failure_as_missing_value_: " << treat_failure_as_missing_value_ << std::endl; 
+    }
```

Comment:
You can remove "mpi::rank() == 0"
This should be handled at the calling site of this function.

Reply:
Sorry still getting used to this! I have tried to resolve this in the last commit, but it doesn't seem to show you that inline...